src/node.cc

```diff
@@ -2446,6 +2455,31 @@ void ProcessEmitWarning(Environment* env, const char* fmt, ...) {
   f.As<v8::Function>()->Call(process, 1, &arg);
 }
 
+void ProcessEmitDeprecationWarning(Environment* env,
+                                   const char* warning,
+                                   const char* deprecation_code) {
+  HandleScope handle_scope(env->isolate());
+  Context::Scope context_scope(env->context());
+
+  Local<Object> process = env->process_object();
+  MaybeLocal<Value> emit_warning = process->Get(env->context(),
+      FIXED_ONE_BYTE_STRING(env->isolate(), "emitWarning"));
```

Comment:
I would probably move the `FIXED_ONE_BYTE_STRING`s here into the list in `env.h`?